tasks/create_target_vm/03_hosted_engine_final_tasks.yml

```diff
@@ -311,6 +311,16 @@
   - debug: var=mresult
   - name: Wait for the engine to come up on the target VM
     block:
+      - name: Make sure http/https is open in firewalld on Engine VM
```

Comment:
This task and task in the file above, both need to be present for successful deployment on RHV 4.4.1 latest build. Without these for some reason engine setup `       - name: Check engine VM health ` task fails in this tasks file. This was not required in past but probably needed now? Currently when I checked services that are allowed in FirewallD on the fresh Engine were 
```
firewall-cmd --list-services
cockpit dhcpv6-client ssh
```
And Ports were blank:
```
firewall-cmd --list-ports

<No output>
```
So adding this task helped me fix the issue.